app/views/brexit_landing_page/_brexit_finders.html.erb

```diff
@@ -1,10 +1,10 @@
 <div class="govuk-grid-row">
   <div class="govuk-body govuk-grid-column-two-thirds">
     <%= render "govuk_publishing_components/components/heading", {
-      text: t('brexit_landing_page.all_brexit_information'),
+      text: t('brexit_landing_page.topic_section_header'),
```

Comment:
In terms of renaming keys for where they're used, I think we normally avoid this because things can get out of date if we, for example, change the component we're using, move things around on the page, or find the same string can be used in multiple places.

... on the other hand, perhaps this doesn't apply here because the text on this page changes so frequently. Up for debate I guess, and doesn't need to be resolved now.

Reply:
yeah, i figured this is also with an eye to this becoming the transition page.
But good to know, i still feel like this could reduce potential for errors, but perhaps that's the specificity of this page and time?